Sources/SwifterSwift/UIKit/UIButtonExtensions.swift

```diff
@@ -184,6 +184,40 @@ public extension UIButton {
         contentEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: insetAmount)
     }
 
+    /// SwifterSwift: Center vertically title text and image on UIButton
```

Comment:
You don't need to mention `UIButton` in the comment for a function in `UIButton`.

Reply:
My english is not good. How about this new one: `alignTextAndImageVertically(imageAboveText:)`?The comments are copied from the previous method. You are very reasonable. I will modify it in the next commit.